### ISSUE TITLE
New restricted files for WordPress, Drupal and Joomla

### ISSUE THREAD

**Issue by azurit** [SpiderLabs/owasp-modsecurity-crs#1531]
These should be added to restricted-files.data for WordPress:
wp-config.php~ (vim backup file)
wp-config.php.swp (vim swap file)

,for Drupal:
/sites/default/default.settings.php~
/sites/default/settings.php~
/sites/default/settings.local.php~
/sites/default/default.settings.php.swp
/sites/default/settings.php.swp
/sites/default/settings.local.php.swp

and for Joomla:
/configuration.php
/configuration.php~
/configuration.php.swp

**Comment by fgsch** [comment]
.swp is now covered. This leave us with:

```
WordPress:
wp-config.php~

Drupal:
/sites/default/default.settings.php~
/sites/default/settings.php~
/sites/default/settings.local.php~

Joomla:
/configuration.php~
```

**Comment by azurit** [comment]
Sending more:
```
# Prestashop
/config/settings.inc.php
# OpenCart + phpBB
/config.php
# OpenCart
/admin/config.php

```

**Comment by fgsch** [comment]
We could exclude the .php~ extension as well but vim backup is not limited to a particular extension.

**Comment by azurit** [comment]
I think that everything, which ends with a ~ is not supposed to be accessible via HTTP(S).

**Comment by theMiddleBlue** [comment]
The problem with ~ is to handle bypass like  `/foo.php~/bar/` and prevent blocking userdir `/~user/`

**Comment by theMiddleBlue** [comment]
ok, I think I got it. Today I'll send a PR.

**Comment by fgsch** [comment]
I believe this is now addressed by 920500. Can you confirm so we can close this issue?

**Comment by fgsch** [comment]
Addressed by #1591 .

**Comment by azurit** [comment]
I'm sorry, i'm running out of time :( Looks good, thank you!
